doc/en/weechat_plugin_api.en.adoc

```diff
@@ -10437,7 +10437,7 @@ def hook_print(buffer: str, tags: str, message: str, strip_colors: int, callback
 
 # example
 def my_print_cb(data: str, buffer: str, date: str, tags: str, displayed: int, highlight: int, prefix: str, message: str) -> int:
-    if int(highlight):
```

Comment:
This cast was to be compatible with WeeChat < 1.0 where the `highlight` parameter was string instead of integer (this was a bug), so this makes code compatible with all WeeChat version (> 0.3.0).

But since 1.0 was released in August 2015, I agree we can now remove this.

Reply:
Ah, okay.

I noticed that `remaining_calls` in the `hook_timer` callback is a string even though it's an integer in the C callback. Is there any reason for this, or is that also a bug?

Reply:
I see no reason why this is a string in scripting API only… this was introduced in 2009 and is a bug for me.

Please keep `str` for now, I'll fix in a separate commit once this PR is merged.